src/main/java/com/artipie/api/ArtipieApi.java

```diff
@@ -62,6 +63,7 @@
  * Artipie API endpoints.
  * @since 0.6
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
+ * @checkstyle ClassFanOutComplexityCheck (500 lines)
```

Comment:
@olenagerasimova I'd suggest to write a puzzle about reducing this class complexity

Reply:
@olegmoz this class already has a puzzle to decompose constructor https://github.com/artipie/artipie/issues/464, I think that's enough considering the fact that it has no methods, ctor only.

Reply:
@olenagerasimova ah, I see. That's clearly enough, thanks